.github/ISSUE_TEMPLATE/04_core_feature_request.yml

```diff
@@ -0,0 +1,62 @@
+name: Core Feature Request
```

Comment:
If this is intended as something internal, maybe we should annotate that here?   Do we see a desire for customers to be making direct requests for Core features?